feature/feature.go

```diff
@@ -28,6 +28,11 @@ const (
 	// // alpha: v1.X
 	// MyFeature featuregate.Feature = "MyFeature".
 
+	// ROSA is used to enable ROSA support
+	// owner: @enxebre
+	// alpha: v2.2
+	ROSA featuregate.Feature = "ROSA"
```

Comment:
Usually we keep adding to this list instead of adding at the top. :D This looks weird. :D Can you please shift it down? :)